### PR TITLE
chapter_8

### PR DESCRIPTION
**Pull request by starocketeer**

### CODE REVIEW

**Comment by starocketeer** [comment]
Ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -2,24 +2,31 @@ class UsersController < ApplicationController
...
   end
+<<<<<<< HEAD
     
   private
     
+=======
+
+  private
+
+>>>>>>> 9030eab... chapter_8
```
Comment (manhnt-0960) [inline app/controllers/users_controller.rb]: please fix it

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module SessionsHelper
+    def log_in user
```
Comment (manhnt-0960) [inline app/helpers/sessions_helper.rb]: indent 2 spaces

File: config/routes.rb [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
-  get   "/help", to: "static_pages#help"
-  get   "/about", to: "static_pages#about"
-  get   "/contact", to: "static_pages#contact"
-  get   "/signup", to: "user#new"
+  get "/home", to: "static_pages#home"       
+  get "/help", to: "static_pages#help"
+  get "/about", to: "static_pages#about"
+  get "/contact", to: "static_pages#contact"
+  get "/signup", to: "user#new"
+  get   "/login", to: "sessions#new"
```
Comment (manhnt-0960) [inline config/routes.rb]:
```suggestion
  get "/login", to: "sessions#new"
```

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (starocketeer) [reply app/controllers/users_controller.rb]: Done

File: app/helpers/sessions_helper.rb (hunk 2 above)
Comment (starocketeer) [reply app/helpers/sessions_helper.rb]: Done

File: config/routes.rb (hunk 3 above)
Comment (starocketeer) [reply config/routes.rb]: Done

**Comment by starocketeer** [comment]
Ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**